### ISSUE TITLE
fill(none) fails on a grayscale image

### ISSUE THREAD

**Issue by vgrigoruk** [cshum/imagor#421]
When I try to apply a `fill(none)` filter to a grayscale image (attached), I receive the following error:
```
{"message":"linear: vector must have 1 or 2 elements","status":406}
```

Could be reproduced easily using the URL below, which uses the image attached to this issue (assuming that imagor is running locally on port 8000): 
```
http://localhost:8000/unsafe/fit-in/200x200/filters:fill(none)/https://github.com/cshum/imagor/assets/487261/88429e96-4bae-4ebb-a7a5-005e070319f6

```

![gray](https://github.com/cshum/imagor/assets/487261/88429e96-4bae-4ebb-a7a5-005e070319f6)

**Comment by vgrigoruk** [comment]
Looks similar to https://github.com/lovell/sharp/issues/411

**Comment by cshum** [comment]
fixed in v1.4.11
